### PR TITLE
✨ Add launch-page

### PR DESCRIPTION
**Pull request by arassaria**
# Description

Add launchpage with outstanding logo animation 🚀 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/pages/Launch.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    "title"
+    "logo";
+`;
+
+const Title = styled.h1`
+  grid-area: title;
+  place-self: end center;
+  padding-bottom: 2rem;
+  font-size: 3rem;
+  color: var(--white);
```
Comment (kschmitz123) [inline client/src/pages/Launch.js]: I would rename this variable to something less specific

**Review by kschmitz123 — APPROVED** [review]
Cool animation 👍

File: client/src/pages/Launch.js (hunk 1 above)
Comment (arassaria) [reply client/src/pages/Launch.js]: Replaced with primary and secondary 🚀

**Review by bwnstck — APPROVED** [review]
so sexy 👩‍🎤

**Review by BelaM80 — COMMENTED** [review]
Lgtm! 🎉🥳

**Pull request merged**